apps/test/unit/templates/sectionProgressV2/IconKeyTest.jsx

```diff
@@ -38,6 +29,15 @@ describe('IconKey Component', () => {
     expect(screen.getByText('Validated')).to.exist;
   });
 
+  it('expands collapses on click', () => {
```

Comment:
I am a big fan of making tests completely independent of order, especially because test order is not guaranteed. I think both of your other options would work great though. 

Do we need a test that it will start collapsed if the setting is false?

Reply:
Sounds good - I modified it based on these thoughts and did add a test to check when it starts collapsed.